### DISCUSSION TITLE
Rustc version to build-native?

### DISCUSSION THREAD

**Discussion by AlexMoutonNoble** [parcel-bundler/parcel#6824] [Q&A] [answered]
Hi
Took a shot at building locally to make PR and having rustc compile issues.
a dependency swc-project (and maybe others?) failing about an or-patterns language feature that does not seem available in fairly recent rust toolchains, (and rustup update nightly doesnt help)

I am using latest darwin nix packages to start. rustc 1.52.1?

Whats a version of rust that does compile?

Thanks
A

**Comment by mischnic** [comment, marked as answer]
It works for me with rustc 1.54.0

**Comment by AlexMoutonNoble** [comment]
Fwiw. I was able to build via nix using nixpkgs-unstable as of today.  rustc 1.53.0
